enrichment_method.py

```diff
@@ -56,13 +58,11 @@ def lambda_handler(event, context):
     # Set up logger.
     current_module = "Enrichment - Method"
     error_message = ''
-    logger = general_functions.get_logger()
 
     bpm_queue_url = None
 
     run_id = 0
     try:
-        logger.info("Starting Enrichment Method")

```

Comment:
Instead of removing the starting logger should we treat this as a similar case as the run_id variable and set the needed variables before the marshmallow test so we can call the get logger at the start?

Not sure on your thoughts.